ci/pipelines/deploy-to-perf.yml

```diff
@@ -0,0 +1,161 @@
+definitions:
+  aws_assumed_role_creds: &aws_assumed_role_creds
+    AWS_ACCESS_KEY_ID: ((.:role.AWS_ACCESS_KEY_ID))
+    AWS_SECRET_ACCESS_KEY: ((.:role.AWS_SECRET_ACCESS_KEY))
+    AWS_SESSION_TOKEN: ((.:role.AWS_SESSION_TOKEN))
+
+  wait_for_deploy_params: &wait_for_deploy_params
+    <<: *aws_assumed_role_creds
+    APPLICATION_IMAGE_TAG: ((.:application_image_tag))
+    NGINX_IMAGE_TAG: ((.:nginx_image_tag))
+    TELEGRAF_IMAGE_TAG: ((.:telegraf_image_tag))
+    ENVIRONMENT: test-perf-1
+
+  deploy_params: &deploy_params
+    <<: *aws_assumed_role_creds
+    APPLICATION_IMAGE_TAG: ((.:application_image_tag))
+    NGINX_IMAGE_TAG: ((.:nginx_image_tag))
+    TELEGRAF_IMAGE_TAG: ((.:telegraf_image_tag))
+    ACCOUNT: test
+    ENVIRONMENT: test-perf-1
+
+  aws_test_config: &aws_test_config
+    aws_access_key_id: ((readonly_access_key_id))
+    aws_secret_access_key: ((readonly_secret_access_key))
+    aws_session_token: ((readonly_session_token))
+    aws_role_arn: arn:aws:iam::((pay_aws_test_account_id)):role/concourse
+    aws_ecr_registry_id: "((pay_aws_test_account_id))"
+    aws_region: eu-west-1
+
+resources:
+  - name: deploy-to-perf-pipeline-definition
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-ci
+      branch: master 
+      paths:
+        - ci/pipelines/deploy-to-perf.yml
+  - name: pay-ci
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-ci
+      branch: master
+  - name: pay-infra
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-infra
+      branch: master
+      username: alphagov-pay-ci
```

Comment:
I realise we're using this `username` all over the place, but it doesn't exist anymore, right? Should it be `alphagov-pay-ci-concourse` ?

Reply:
Yeah it should be `alphagov-pay-ci-concourse`. I think our pipeline are still working because it's using a cache when getting the pay-infra resource.
```
INFO: found existing resource cache
```

Reply:
Fixed